src/glue/auth_handler.cpp

```diff
@@ -122,6 +122,29 @@ std::vector<std::vector<memgraph::query::TypedValue>> ShowRolePrivileges(
 }
 
 #ifdef MG_ENTERPRISE
+std::vector<std::vector<memgraph::query::TypedValue>> ShowDatabasePrivileges(
+    const std::optional<memgraph::auth::User> &user) {
+  if (!memgraph::license::global_license_checker.IsEnterpriseValidFast() || !user) {
+    return {};
+  }
+
+  const auto &db = user->db_access();
+  const auto &allows = db.GetAllowAll();
+  const auto &grants = db.GetGrants();
+  const auto &denies = db.GetDenies();
+
+  std::vector<memgraph::query::TypedValue> res;  // First element is a list of granted databases, second of revoked ones
```

Comment:
This is just the result that gets sent to the user.
It has to be a vector of TypedValue; the TypedValue here will be a vector of 2 elements.
I'll check in other queries for a better solution.

Reply:
Other Show*(user) and Get*(username) functions of the AuthHandler directly return the TypedValue vector. 
I think it would be better to follow the same template. This gets called from the interpreter and its result is directly used.
Let me know if that's okay with you.

Reply:
Sure, outside the scope of this PR.